allauth/socialaccount/providers/garminconnect/client.py

```diff
@@ -0,0 +1,52 @@
+import requests
+from urllib.parse import parse_qsl
+
+from django.utils.translation import gettext as _
+from requests_oauthlib import OAuth1
```

Comment:
Question: this provider is based on OAuth 1.0 -- yet, reading over here https://developer.garmin.com/connect-iq/core-topics/authenticated-web-services/ it seems as if they also support OAuth 2.0. If that is the case, shouldn't 2.0 instead of 1.0 be integrated?